zookeeper-server/src/test/java/org/apache/zookeeper/test/QuorumUtil.java

```diff
@@ -97,7 +101,7 @@ public QuorumUtil(int n, int syncLimit) throws RuntimeException {
             N = n;
             ALL = 2 * N + 1;
             tickTime = 2000;
-            initLimit = 3;
+            initLimit = 5;
```

Comment:
why are you changing this value ?

Reply:
It is unnecessary, doesn't make any difference, thanks for noticing it.
I just committed this by accident, I will revert.